src/components/views/elements/ReplyChain.tsx

```diff
@@ -111,10 +111,8 @@ export default class ReplyChain extends React.Component<IProps, IState> {
         // can't just rely on ev.getContent() by itself because historically we
         // still show the reply from the original message even though the edit
         // event does not include the relation reply.
```

Comment:
This comment no longer applies

Reply:
Also `get replyEventId` in event.ts in js-sdk doesn't seem to honor the will of this comment and uses `getWireContent()` instead